packages/wagmi/src/connectors/EIP6963Connector.ts

```diff
@@ -50,6 +51,17 @@ export class EIP6963Connector extends InjectedConnector {
     return data
   }
 
+  protected override onAccountsChanged = (accounts: string[]) => {
```

Comment:
Are we sure that we definitely have an array of strings as a parameter? I couldn't find a source where this event was triggered. If it's in case that it could be `undefined` or something else, the `accounts.length` or `accounts[0]` property access' might fail.

Could you share with me some links where I can find more context about the EIP-6963? I have the link below. More would be appreciated.

- https://eips.ethereum.org/EIPS/eip-6963

Reply:
Hi, this comes from EIP-1193 standard
https://eips.ethereum.org/EIPS/eip-1193

What I basically did here was copy/paste the function from Wagmi's injectedConnector and add the line to remove the custom storage item.